Specify the insert order on SQL query at `Task230119MigrateContentToProperPersonaTagAndRemoveDupTags`

### Parent Issue

_No response_

### Problem Statement

A client is facing an issue when try to update from `23.01.3` to `23.01.4`. 
Here are the logs that correspond to it:
```
12:28:08.677  INFO  runonce.Task230119MigrateContentToProperPersonaTagAndRemoveDupTags - ==> Executing upgrade script
12:28:08.677  INFO  runonce.Task230119MigrateContentToProperPersonaTagAndRemoveDupTags - ======================================================================
12:28:08.677  INFO  db.DotConnect - Executing INSERT INTO tag_inode
SELECT tag.tag_id, aux.inode, aux.field_var_name, aux.mod_date
FROM (SELECT tag_inode.*, tag.tagname
      FROM tag_inode
               JOIN tag ON tag_inode.tag_id = tag.tag_id
      WHERE tagname LIKE '%:persona') aux,
     tag
WHERE tag.tagname = REPLACE(aux.tagname, ':persona', '') AND NOT EXISTS (SELECT 1 FROM tag_inode WHERE tag.tag_id = tag_id AND aux.inode = inode)
com.dotmarketing.exception.DotDataException: ERROR: column "mod_date" is of type timestamp with time zone but expression is of type character varying
  Hint: You will need to rewrite or cast the expression.
  Position: 53
        at com.dotmarketing.startup.AbstractJDBCStartupTask.executeUpgrade(AbstractJDBCStartupTask.java:793)
        at com.dotmarketing.startup.StartupTasksExecutor.executeBackportedTasks(StartupTasksExecutor.java:256)
      
Caused by: org.postgresql.util.PSQLException: ERROR: column "mod_date" is of type timestamp with time zone but expression is of type character varying
  Hint: You will need to rewrite or cast the expression.
  Position: 53
```

The reason for this problem is that the columns in the `tag_inode` table are ordered differently for the current client. The issue is only affecting this particular client at the moment. To prevent this problem from occurring with other clients and to resolve it for affected clients, the insert query at [Task230119MigrateContentToProperPersonaTagAndRemoveDupTags](https://github.com/dotCMS/core/blob/master/dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task230119MigrateContentToProperPersonaTagAndRemoveDupTags.java) needs to be adjusted to specify the order of the fields that are being inserted.


### Steps to Reproduce

1. Restore the database of the client on your local machine.
2. Try to run dotcms on `23.01.4` using the database of the client.
3. You should be able to see the error on the logs.

### Acceptance Criteria

The insert query needs to specify the order of the fields that are being inserted.

### dotCMS Version

23.01.4

### Proposed Objective

Customer Support

### Proposed Priority

Priority 2 - Important

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

_No response_

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### Sub-Tasks & Estimates

_No response_

--- a/dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task230119MigrateContentToProperPersonaTagAndRemoveDupTags.java
+++ b/dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task230119MigrateContentToProperPersonaTagAndRemoveDupTags.java
@@ -22,7 +22,7 @@ public class Task230119MigrateContentToProperPersonaTagAndRemoveDupTags extends 
             + "AND t.tagname LIKE '%:persona';";
     final String DELETE_BAD_RELS_MSSQL = "DELETE ti FROM tag_inode ti JOIN tag ON ti.tag_id = tag.tag_id\n"
             + "WHERE tag.tagname LIKE '%:persona';";
-    final String SQL = "INSERT INTO tag_inode\n"
+    final String SQL = "INSERT INTO tag_inode (tag_id, inode, field_var_name, mod_date)\n"
             + "SELECT tag.tag_id, aux.inode, aux.field_var_name, aux.mod_date\n"
             + "FROM (SELECT tag_inode.*, tag.tagname\n"
             + "      FROM tag_inode\n"

